test/ibmq/test_experiment.py

```diff
@@ -400,14 +409,14 @@ def test_delete_experiment(self):
 
     def test_reference_experiment_analysis_results(self):
         """Test referencing analysis results in an experiment."""
-        ref_result = self.provider.experiment.analysis_results()[0]
+        ref_result = self.provider.experiment.analysis_results(tags=['qiskit-test'])[0]
```

Comment:
Similar comment about having a help function to list analysis results that by default includes that tag.

Reply:
Updated!